benchmarks/operator_benchmark/pt/qlinear_test.py

```diff
@@ -45,17 +47,31 @@ def init(self, N, IN, OUT):
         qW = torch.quantize_per_tensor(W, scale=scale, zero_point=0, dtype=torch.qint8)
 
         self.input = qX
-        self.qlinear = nnq.Linear(IN, OUT)
+
+        # Assume that the `self.qlinear` is set in the child
         self.qlinear.weight = qW
         self.qlinear.scale = scale
         self.qlinear.zero_point = zero_point
```

Comment:
I have a uncommitted PR: https://github.com/pytorch/pytorch/pull/24059.

For qconv2d op, we have scale and zero points as torch.tensor (https://github.com/pytorch/pytorch/blob/master/benchmarks/operator_benchmark/pt/qconv_test.py#L75-L76). However, for qlinear op, we have these quantization parameters as the pure number.

There are some back-and-forth changes here. Please advise what is the current status. I see at least there is mismatch between qlinear and qconv.

Reply:
The benchmark code will not have the unit test. So it is likely any code you push here for this file will not have the unit test coverage (Correct me if I am wrong). If so, please make sure you can locally run these benchmarks.

Reply:
I think for the linear it is still accepting the non-tensor variables